src/pages/Feedback/Feedback.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useEffect } from 'react'
+import { Row, Container } from 'react-bootstrap'
+import FeedbackForm from '@/forms/Feedback'
+import FeedbackApi from '@/API/feedback'
+import Item from './components/Item'
+import ContentBox from '@/components/ContentBox/ContentBox'
+import alert from '@/utils/alert'
+
+function getRecipientObj(recipient, recipients) {
+    const obj = recipients.find((rec) => rec.name === recipient)
+    return obj
+}
+
+export default function Feedback() {
+    const [recipients, setRecipients] = useState([])
+    const [recipient, setRecipient] = useState('')
+
+    useEffect(() => {
+        async function FetchApi() {
+            const response = await FeedbackApi.getAll()
+            setRecipients(response?.data)
+            setRecipient(response?.data[0].name)
+        }
+        FetchApi()
+    }, [])
+
+    async function sendRequest(values) {
+        const data = { category: values.category, text: values.text }
+        if (values.email) data.email = values.email
+
+        if (await FeedbackApi.post(getRecipientObj(recipient, recipients)?.name, data)) {
+            alert('Вы успешно отправили свой запрос', {
+                title: 'Успех!',
+            })
+        }
+    }
+
+    return (
+        <Container className="my-5">
+            <ContentBox header="Шаг 1. Выберите, кому будет направлен ваш вопрос" className="mb-5">
+                <Row
+                    className="gy-4"
```

Comment:
На мобильных разрешениях великоваты отступы
Давай сократим:
```suggestion
                    className="gy-3"
```